Source/Plugins/EditorModules/ProjectView/Modules/ProjectFolderView.cs

```diff
@@ -576,6 +588,27 @@ protected bool DisplayConfirmDeleteSelectedFiles()
 				MessageBoxIcon.Question);
 			return result == DialogResult.Yes;
 		}
+		protected void DisplayFileImportDialog()
+		{
+			OpenFileDialog dialog = new OpenFileDialog
+			{
+				Multiselect = true,
+				InitialDirectory = this.importSourcePath ?? Path.GetFullPath(DualityApp.DataDirectory)
```

Comment:
Please set the title of the dialog to something appropriate - maybe "Select File to Import" or something.